licenses.json

```diff
@@ -2,5 +2,6 @@
   "GPL-2.0-or-later": "GPLv2+",
   "GPL-3.0": "GPLv3",
   "GPL-3.0+": "GPLv3+",
-  "GPLv3-or-later": "GPLv3+"
+  "GPLv3-or-later": "GPLv3+",
```

Comment:
Perhaps this should be a separate commit.

Reply:
it can be, if you want to. but still noting that my addition is a valid spdx.

Reply:
Yes. I think it should be a separate commit. I'd be OK with just adding a line but let's keep it sorted.

Reply:
updated